packages/mdc-menu/README.md

```diff
@@ -136,6 +136,25 @@ classes:
 | `mdc-simple-menu--open-from-bottom-right` | Open the menu from the bottom right. |
 
 
+#### Disabled list items
```

Comment:
Shall we provide a method in menu to set menu item to disabled programmatically? 
I think we could potentially use the ones that defined in `mdc-select`, couldn't we?